airflow/jobs/scheduler_job.py

```diff
@@ -1318,6 +1284,7 @@ def _enqueue_task_instances_with_queued_state(self, simple_dag_bag,
                 command,
                 priority=priority,
                 queue=queue)
+        return i+1
```

Comment:
```suggestion
```
This method doesn't have to return anything. The user of this method can call len () on one parameter of this method. This will always give the same result. 

> Explicit is better than implicit.

Reply:
I'll double check, but I think this avoided a `count()` query when we already new the answer!

Reply:
In the previous version, we use len() on list. It doesn't look like it needs optimization.
https://github.com/apache/airflow/pull/6792/files/59fbc2f84a96c1677c8e420d7ce761f1d82b06ac#diff-c35269bcfbbe386e269ffa7487e86192L1355

Reply:
Hmmm yes. Maybe this changed in the rebasing, or I'm making stuff up :grin: